clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/AggKafkaConsumerServiceStats.java

```diff
@@ -0,0 +1,123 @@
+package com.linkedin.davinci.stats;
+
+import com.linkedin.venice.exceptions.VeniceException;
+import com.linkedin.venice.meta.ReadOnlyStoreRepository;
+import com.linkedin.venice.stats.AbstractVeniceAggStoreStats;
+import com.linkedin.venice.stats.StatsSupplier;
+import com.linkedin.venice.utils.SystemTime;
+import io.tehuti.metrics.MetricsRepository;
+import java.util.function.LongSupplier;
+
+
+/**
+ * This class is an aggregate place that keeps stats objects for multiple stores and total stats for each region for
+ * AggKafkaConsumerService.
+ *
+ * For total stats for a given region, use this class to record stats. For store-level stats, delegate them to
```

Comment:
Good Java doc, this is critical for me to understand the complicated stats class relationship :)